build/release/collect-licenses.sh

```diff
@@ -43,12 +43,14 @@ if [ "${SRC}" = "-h" ]; then
 	exit 0
 fi
 
-for jar_file in $(find -L "${SRC}" -name "*.jar" | grep -v "kyuubi-")
+for jar_file in $(find -L "${SRC}" -name "*.jar")
 do
-	DIR="${TMP}/$(basename -- "${jar_file}" .jar)"
-	mkdir -p "${DIR}"
-	JAR=$(realpath "${jar_file}")
-	(cd "${DIR}" && jar xf ${JAR} META-INF/NOTICE META-INF/licenses)
+  if [[ $(basename $jar_file) != kyuubi-* ]]; then
```

Comment:
Would you please also replace all `tab` in this script with `2 spaces`?
BTW, please keep the same code style with original content, e.g. `${jar_file}`, and quote string by `"`